py3status/modules/volume_status.py

```diff
@@ -16,9 +16,12 @@
         (default 0)
     cache_timeout: how often we refresh this module in seconds.
         (default 10)
-    channel: Alsamixer channel to track.
+    channel: Alsamixer channel to track (ignored by pulseaudio)
         (default 'Master')
-    device: Alsamixer device to use.
+    command: Choose between "amixer" and "pamixer". If None, try to guess based
+        on available commands.
+        (default None)
+    device: Device to use.
         (default 'default')
```

Comment:
Ok (you're talking about `device`, right?). This way the default is backend dependant. I'll do the same for `channel`.